tools/RELEASE.md

```diff
@@ -154,7 +156,12 @@ and librdkafka versions:
     py: ('0.11.4', 721920) c: ('0.11.4-RC1', 722121)
 
 
-## 8. Upload packages to PyPi
+## 9. Merge RC branch into master
+
+Once all builds are successful and artifacts have been validated proceed with merging the changes into Master. 
+Be sure to squash any unrelated commits (typos, build errors) to keep the master commit log clean. 
```

Comment:
You typically squash/fixup **related** commits.

So maybe something like:
`Be sure to tidy up the commit log in the PR branch by squashing related commits (typos, build fixes, etc)`.

Reply:
Thanks I was having trouble figuring out how to correctly word that.